### DISCUSSION TITLE
获取某个群的全部成员账号信息，查询机器人的全部好友。

### DISCUSSION THREAD

**Discussion by 930893138** [ForteScarlet/simpler-robot#459] [不要在这里发帖子了]
多次查询文档没有找到类似的api，
例：
1.查找某个群的全部成员信息
2.查询这个机器人的全部好友。
期望告知一下！

**Comment by ForteScarlet** [comment]
版本是什么，2.x与3.x的方式是不同的

**Comment by 930893138** [comment]
2.x




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "simple-robot/simpler-robot"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月25日(星期天) 下午2:35
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [simple-robot/simpler-robot] 获取某个群的全部成员账号信息，查询机器人的全部好友。 (Discussion #459)





 
版本是什么，2.x与3.x的方式是不同的

**Reply by ForteScarlet** [reply]
可以试着参考 [2.x文档 送信器 Getter](https://www.yuque.com/simpler-robot/simpler-robot-doc/gsbknt#CecwJ)
